tests/testdata/qgis_server/getcapabilities-map.txt

```diff
@@ -199,6 +199,54 @@ Content-Type: text/xml; charset=utf-8
      </LegendURL>
     </Style>
    </Layer>
+   <Layer queryable="1">
+    <Name>&lt;test layer name></Name>
+    <Title>&lt;test layer name></Title>
+    <Abstract>A test vector layer with tag chars without title</Abstract>
+    <CRS>CRS:84</CRS>
+    <CRS>EPSG:4326</CRS>
+    <CRS>EPSG:3857</CRS>
+    <EX_GeographicBoundingBox>
+     <westBoundLongitude>8.203459</westBoundLongitude>
+     <eastBoundLongitude>8.203547</eastBoundLongitude>
+     <southBoundLatitude>44.901394</southBoundLatitude>
+     <northBoundLatitude>44.901483</northBoundLatitude>
+    </EX_GeographicBoundingBox>
+    <BoundingBox maxy="5606025.238" maxx="913214.675" miny="5606011.456" CRS="EPSG:3857" minx="913204.912"/>
+    <BoundingBox maxy="8.203547" maxx="44.901483" miny="8.203459" CRS="EPSG:4326" minx="44.901394"/>
+    <Style>
+     <Name>default</Name>
+     <Title>default</Title>
+     <LegendURL>
+      <Format>image/png</Format>
+      <OnlineResource xlink:type="simple" xlink:href="https://www.qgis.org/?*****" xmlns:xlink="http://www.w3.org/1999/xlink"/>
+     </LegendURL>
+    </Style>
+   </Layer>
+   <Layer queryable="1">
+    <Name>&lt;test layer title></Name>
+    <Title>&lt;title></Title>
+    <Abstract>A test vector layer with tag chars with title</Abstract>
```

Comment:
```suggestion
    <Abstract>A test vector layer with &lt;tag&gt; chars with title</Abstract>
```
maybe test abstract too...

Reply:
Done in https://github.com/qgis/QGIS/pull/55751/commits/4763dd017071b770eb93991f9ba0161d3bca2ca4 (as outlined above, when saving a QGIS project with `<` `>`, only the opening tag char is replaced for the XML).